workspaces/blackduck/plugins/blackduck-common/src/types.ts

```diff
@@ -93,3 +93,11 @@ export type BD_VERSIONS_API_RESPONSE = {
   appliedFilters: [];
   _meta: META;
 };
+
+/**
+ * @public
+ */
+export type BD_CREATE_PROJECT_API_RESPONSE = {
```

Comment:
shouldn't this go in the `node` package?
Probably the same should go for all the types defined here, unless they are used by the frontend package